docs/first-steps/post-deployment.md

```diff
@@ -176,6 +176,91 @@ leverage credentials configure --fetch-mfa-device --type SECURITY
 !!! note
     If a layer was already set with BOOTSTRAP credentials, when changing the credential type Terraform has to be reconfigured: `leverage tf init -reconfigure`.
 
+
+## Add New AWS Accounts
+
+If  necessary you can easily add new AWS accounts to your Leverage project by following these steps:
+
+!!! info ""
+    1. Go to `management/global/organizations`.
+    2. Edit the `locals.tf` file to add/remove accounts from the local `accounts` variable.
+    3. Finally, run the [Terraform workflow](https://leverage.binbash.com.ar/user-guide/ref-architecture-aws/workflow/) to apply the new changes.
+    4. Add the new account to the `config/common.tfvars` file.
+
+From here, to ensure it's properly integrated, you may very likely want to create the initial directory structure for this new account *as explained right below*.
+
+
+#### Create the initial directory structure for a new account
+  As an example, we will set up the `apps-prd` account by using the `apps-devstg` as a source reference code:
+
+!!! info ""
+    1. Ensure you are at the root of this repository
+    2. Create the initial directory structure for the new account:
+        ```
+        mkdir -p apps-prd/{global,us-east-1}
+        ```
+    3. Set up the config files:
+        1. Create a config files for this account: `cp -r apps-devstg/config apps-prd/config`
+        2. Open `apps-prd/config/backend.tfvars` and replace any occurrences of `devstg` with `prd`
+        3. Do the same with `apps-prd/config/account.tfvars`
+        4. Open up `apps-prd/config/backend.tfvars` again and replace this:
+            ```
+            profile = "bb-apps-prd-devops"
+            ```
+            with this:
+            ```
+            profile = "bb-apps-prd-oaar"
+            ```
+        5. In the step above, we are switching to the OAAR (OrganizationalAccountAccessRole) role because we are working with a brand new account that is empty, so, the only way to access it programmatically is through the OAAR role.
+        6. Now it's time to configure your OAAR credentials (if haven't already done so). For that you can follow the steps in [this section](https://leverage.binbash.com.ar/first-steps/management-account/#update-the-bootstrap-credentials) of the official documentation.
+    4. Create the `base-tf-backend` layer:
+        1. Copy the layer from an existing one: `cp -r apps-devstg/us-east-1/base-tf-backend apps-prd/us-east-1/base-tf-backend`
+        2. Go to the `apps-prd/us-east-1/base-tf-backend` directory and open the `config.tf` file. Comment this block:
+            ```
+            backend "s3" {
+            key = "apps-devstg/tf-backend/terraform.tfstate"
+            }
+            ```
+        3. Now run the [Terraform workflow](https://leverage.binbash.com.ar/user-guide/ref-architecture-aws/workflow/) to initialize and
+           apply this layer.  (You may need to pass the `--skip-validation` flag to
+           `leverage tf init`.)
```

Comment:
```suggestion
           `leverage tf init`).
```

Reply:
Actually I think you may want to drop the parenthesis otherwise the period after the closing parenthesis will not be finishing a paragraph. For instance: `My paragraph here. (This between parenthesis).`
Or you may also choose to move the phrase between parenthesis at the end of the previous paragraph like so: `My paragraph here (this between parenthesis).`

Reply:
Definitely. I didn't realize it was a full sentence inside of the parenthesis. Thanks for pointing it out!